docs/topic/troubleshooting.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting and debugging
+
+These sections describe a few common and useful tips for troubleshooting our infrastructure.
+
+## Roll back (revert) a helm deploy
+
+Sometimes it is useful to simply **revert** a Kubernetes deployment with Helm.
+For example, if you've manually deployed something via `helm install`, and notice that something is wrong with our deployment.
+
+:::{note}
+Ideally, this would have happened automatically via CI/CD, but sometimes a manual deploy is still necessary!
+:::
+
+If you'd simply like to revert back to the state of the Kubernetes infrastructure from before you ran `helm install`, try the following commands:
+
+- **Get the deployment name and revision number for the latest deploy**. To do so, run this command:
+  ```bash
+  helm list -n {{NAMESPACE}}
```

Comment:
```suggestion
  helm list --namespace {{NAMESPACE}}
```

My preference is to use the long flags in documentation so it's clear what is being run and it's easier to find out more about that flag through a web search